recipes/cpr/all/conanfile.py

```diff
@@ -60,9 +75,6 @@ def configure(self):
         if not self._supports_winssl:
             del self.options.with_winssl
 
-        if self._supports_openssl and tools.is_apple_os(self.settings.os):
-            self.options.with_openssl = False # Default libcurl in CCI is `with_ssl="darwin"` which is unclear if cpr supports this
-
```

Comment:
Please add this code back in

Reply:
I am curious: you mention you have `1.5.2` running in production with `openssl`, but this snippet seems to imply it wouldn't have used it anyway. Am I missing something?

Reply:
In the 1.5.2 build files, there's no distinction with what "backend ssl" is being used... enabling OpenSSL on CPR and giving it a CURL with different options works 

I am using

```txt
cpr:with_openssl=True
libcurl:with_ssl="darwinssl"
```

Without any issues

Reply:
Gotcha. I can open a separate PR for these changes if you'd like. I wish the auto-merge didn't get it :(

Reply:
No, I think I will ask CPR for a new release... 1.6.0 does not help me and this change won't affect us but

 thank you very much for the offer!

Reply:
Ah, I was mainly thinking to prevent breaking your 1.5.2 config...